src/components/calcite-scrim/calcite-scrim.scss

```diff
@@ -0,0 +1,40 @@
+:host {
+  display: flex;
+  position: relative;
+  --calcite-scrim-background: #{rgba($blk-000, 0.75)};
+}
+
+// dark theme
+:host([theme="dark"]) {
+  --calcite-scrim-background: #{rgba($blk-200, 0.75)};
```

Comment:
Should this use `$blk-240` which would match what the modal previously had?
Modal was setting the scrim color to `--calcite-modal-scrim: rgba(0, 0, 0, 0.75);`

Reply:
Sure. I'll update. What about the light theme color?

Reply:
Looks good! 💥